content/resources/how-to-guides/state-changes-across-task-executions.md

```diff
@@ -0,0 +1,155 @@
+    |> map( fn: (r) => ({r with
+    _level: if r._value > 40.0 then "crit"  else "ok",
+        }))
+    ```
+
+    Where `states` might look like: 
+
+    | measurement  | tagKey1   | _field    | _value | _level | _time |  
+    | ------------ | --------- | --------- | ------ | ------ | ----- |
+    | _measurement | tagValue1 | fieldKey1 | 30.0   | "ok"   | time1 | 
+    | _measurement | tagValue1 | fieldKey1 | 50.0   | “crit” | time2 | 
+
+
+4. Write the “states” to a bucket. You could also elect to write the data to a new bucket or a new measurement. If you want to write the data to a new measurement, make sure to change the measurement name from your “states” data with the [set()](/flux/v0.x/stdlib/universe/set/) function first. 
+
+    ```js
+    states 
+    |> to(bucket : "<bucket>") 
+    ```
+
+5. Find the latest value from the previous task run and store it in a variable “last_state_previous_task”,
+
+    ```js
+    last_state_previous_task = from(bucket: "<bucket>")
+      |> range(start: experimental.subDuration(d: task.every,
+
+        from: tasks.lastSuccess(orTime: -task.every))
+      |> filter(fn: (r) => r._measurement == "<measurement>")
+      |> filter(fn: (r) => r.tagKey == "<tagValue1>")
+      |> filter(fn: (r) => r._field == "<fieldKey1>")
+      |> last() 
+    ```
+
+    Where `last_state_previous_task` might look like: 
+
+      | measurement  | tagKey1   | _field    | _value | _level | _time |  
+      | ------------ | --------- | --------- | ------ | ------ | ----- |
+      | _measurement | tagValue1 | fieldKey1 | 55.0   | "crit" | time0 | 
+
+6. Union “states” and “last_state_previous_task”. Store this data in a variable “unioned_states”.
+
+    ```js
+    unioned_states = union(tables: [states, last_state_previous_task])
+
+    // use the sort() function to guarantee that you preserve the time order
+
+    |> sort(columns: ["_time"], desc: true)
+    ```
+
+    Where `unioned_states` might look like: 
+
+    | measurement  | tagKey1   | _field    | _value | _level | _time |  
+    | ------------ | --------- | --------- | ------ | ------ | ----- |
+    | _measurement | tagValue1 | fieldKey1 | 55.0   | "crit" | time0 | 
+    | _measurement | tagValue1 | fieldKey1 | 30.0   | "ok"   | time1 | 
+    | _measurement | tagValue1 | fieldKey1 | 50.0   | "crit" | time2 | 
+
+7. Discover state changes in “unioned_states”. Store this data in a variable “state_changes”.
+
+    ```js
+    state_changes = unioned_states 
+    |> monitor.stateChangesOnly()
+    ```
+
+    Where `state_changes` might look like: 
+    | measurement  | tagKey1   | _field    | _value | _level | _time |  
+    | ------------ | --------- | --------- | ------ | ------ | ----- |
+    | _measurement | tagValue1 | fieldKey1 | 30.0   | "ok"   | time1 | 
+    | _measurement | tagValue1 | fieldKey1 | 50.0   | "crit" | time2 | 
+
+8.  Notify on state changes that span across the last two tasks to catch any state changes that occur across task executions.  
```

Comment:
Not sure if this step is needed in this guide, but I think we can reference sending notifications (I'm working on a set of docs that walks through sending notifications for each supported endpoint). But you can leave it in if you want.

Reply:
I think it's helpful to have a complete example so users don't have to navigate to another page to accomplish their goal.